packages/expect/src/__tests__/toThrowMatchers.test.ts

```diff
@@ -231,9 +231,8 @@ matchers.forEach(toThrow => {
       // Received message in report if object has message property.
       class ErrorMessage {
         // not extending Error!
-        constructor(message) {
-          this.message = message;
-        }
+        // @ts-expect-error
+        constructor(private message: string) {}
```

Comment:
I was fixing this one too. Just did not manage to finish yet ;D

```suggestion
        constructor(public message: string) {}
```